src/components/NotificationRow.tsx

```diff
@@ -84,20 +84,6 @@ export const NotificationRow: React.FC<IProps> = ({
           </button>
         </div>
       </div>
-
-      <div className="flex justify-center items-center w-8">
-        <button
-          className="focus:outline-none"
-          title="Mark as Read"
-          onClick={() => markNotification(notification.id, hostname)}
-        >
-          <CheckIcon
-            className="hover:text-green-500"
-            size={20}
-            aria-label="Mark as Read"
-          />
-        </button>
-      </div>
```

Comment:
Why remove the `Mark as Read` button?

I agree that the setting to `Mark as Read on click` is now useless, but IMO we should keep the button as a way of clearing a notification without having to actually open it in Github!

Reply:
Good catch - will correct @adufr